gallery/templates/view_dir.html

```diff
@@ -83,6 +88,13 @@ <h2>This album is empty.</h2>
         <a href="#!" class="btn btn-primary" onclick="editDirDescription()">Edit</a>
         {% if auth_dict['can_edit'] or auth_dict['is_organizer'] %}
         <a href="#!" class="btn btn-warning" onclick="moveDir()">Move</a>
+        {% if directory.parent != None %}
+            {% if directory.pinned %}
+            <a href="#!" class="btn btn-warning" onclick="unpinDir({{ directory.id }})">Unpin</a>
+            {% else %}
+            <a href="#!" class="btn btn-warning" onclick="pinDir({{ directory.id }})">Pin</a>
+            {% endif %}
+        {% endif %}
```

Comment:
this will allow uploaders to see the pin buttons, no?

Reply:
also
curious: why directory.parent != None?

Reply:
Don't want to have the pin button on the root directory, it's pointless